composer.json

```diff
@@ -7,20 +7,21 @@
 		"email": "jason@stallin.gs"
 	}],
 	"require": {
-		"squizlabs/php_codesniffer": "2.8.*",
-		"wimg/php-compatibility": "dev-master",
-		"simplyadmire/composer-plugins" : "@dev"
+		"squizlabs/php_codesniffer": "*",
+		"wimg/php-compatibility": "^8.0"
 	},
 	"require-dev": {
 		"wp-coding-standards/wpcs": "dev-master",
 		"neronmoon/scriptsdev": "^0.1.0"
 	},
 	"scripts": {
-		"post-update-cmd": "vendor/bin/phpcs --config-delete installed_paths; rm -rf vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility",
-		"post-install-cmd": "vendor/bin/phpcs --config-delete installed_paths; rm -rf vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/PHPCompatibility"
+		"post-update-cmd": "vendor/bin/phpcs --config-delete installed_paths; rm -rf vendor/squizlabs/php_codesniffer/src/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/src/Standards/PHPCompatibility",
+		"post-install-cmd": "vendor/bin/phpcs --config-delete installed_paths; rm -rf vendor/squizlabs/php_codesniffer/src/Standards/PHPCompatibility; cp -rp vendor/wimg/php-compatibility vendor/squizlabs/php_codesniffer/src/Standards/PHPCompatibility"
 	},
-	"scripts-dev": {
-		"post-update-cmd": "vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs/;",
-		"post-install-cmd": "vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs/;"
+	"extra": {
+		"scripts-dev": {
+			"post-update-cmd": "vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs/;",
+			"post-install-cmd": "vendor/bin/phpcs --config-set installed_paths vendor/wp-coding-standards/wpcs/;"
```

Comment:
Better to use a Composer plugin for this, as is advised by both WPCS as well as PHPCompatibility. See their respective readme's.